CondFormats/GeometryObjects/interface/PTrackerParameters.h

```diff
@@ -115,6 +115,8 @@ class PTrackerParameters
   TIBItem tib;
   TIDItem tid;
   TOBItem tob;
+
+  std::vector<int> pDetIdShifts;
```

Comment:
@davidlange6 - yes, it is a new schema which is not in ProdDB yet. It will require an action to create one on DB group side. The payload can be tested with PrepDB where this change is ok.